x/incentives/keeper/distribute.go

```diff
@@ -614,7 +630,7 @@ func (k Keeper) distributeInternal(
 				// and inactive gauges. By the time we get here, the gauge should be active.
 				ctx.BlockTime(),
 				// Only default uptime is supported at launch.
-				types.DefaultConcentratedUptime,
+				gaugeUptime,
```

Comment:
This change gives the flexibility of creating external incentives that can incentivize any authorized uptime.

I don't see how this addressed internal incentives that would still continue incentivizing 1ns uptime.

I think the goal was to simply change this value to a 1 min uptime.

Therefore, we need 2 things:
1. Gov prop to expand  CL authorized uptime parameter with 1 min
2. Either hardcode 1min here or, better, make this a configurable parameter

Reply:
Realized I never followed up here – we discussed offline after this message was sent and came to the conclusion that there are a few additional fronts that need to be addressed for uptimes to be enabled safely. It's possible that these will require making some changes to existing gauge logic to avoid leaning on abstractions that weren't intended to be used as generally as they currently are.

Will leave this PR as a draft for now as I continue to work on the broader change (which is taking a bit more design work given how it touches various moving parts)